src/mission.h

```diff
@@ -42,7 +42,8 @@ enum mission_type_id {
     MISSION_JOIN_TRACKER,                  //demon slayer 4b
     MISSION_FREE_MERCHANTS_EVAC_1,         //Clear Back Bay
     MISSION_FREE_MERCHANTS_EVAC_2,         //Kill Raiders
-    MISSION_FREE_MERCHANTS_EVAC_3,         //Acquire Plutonium Cells
+    MISSION_FREE_MERCHANTS_EVAC_3,         //Info from Commune
+    MISSION_FREE_MERCHANTS_EVAC_4,         //Acquire Plutonium Cells
```

Comment:
That looks like it will screw up the ids of existing missions. What was MISSION_OLD_GUARD_REP_1 will now be MISSION_FREE_MERCHANTS_EVAC_4.